.github/workflows/new_version.yml

```diff
@@ -3,9 +3,8 @@
 name: publish
 
 on:
-  create:
-    tags:
-      - v*
+  release:
+    types: [published]
```

Comment:
I don't want to create a release by hand. Just tagging the commit seems way easier

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags looks the way to go then. I will give it a try